runtime/lib.rs

```diff
@@ -140,6 +140,11 @@ pub static UNSTABLE_GRANULAR_FLAGS: &[UnstableGranularFlag] = &[
   },
 ];
 
+pub fn exit(code: i32) -> ! {
+  crate::ops::otel::flush();
+  std::process::exit(code);
```

Comment:
Can you please add `std::process::exit` to disallowed methods in `clippy.toml` files so someone doesn't accidently use it?